modules/cli-administrator-other.adoc

```diff
@@ -38,6 +38,15 @@ $ oc adm config view
 $ oc adm config use-context test-context
 ----
 
+== must-gather
+
+Gather debugging information.
+
+.Example: Gather debugging information
```

Comment:
Consider reviewing https://docs.google.com/document/d/1AN0yobTx9YsnwfGdoGjLywx2gfvEL3U6E8Oy8RgjJFA/edit?ts=5cb47890 for some context as to why you want this.